### DISCUSSION TITLE
What about chromebooks?

### DISCUSSION THREAD

**Discussion by matemaciek** [fhdm-dev/scoppy#100] [Q&A] [answered]
Play store shows that my device is not compatible with Scoppy app, any ideas why? It has regular USB sockets, so connecting the board should work - and if it doesn't, there's always wifi connection.

**Comment by fhdm-dev** [comment, marked as answer]
Excellent question!
It turns out that [apps can't use USB](https://developer.android.com/topic/arc/manifest) on Chromebooks and the Scoppy app requires USB support for it to be installed.

I've been reluctant to remove the USB requirement for two reasons:

1. I can imagine people complaining the app isn't working when in fact their device doesn't support USB host mode (OTG). I should be able to get the app to detect this and show a warning message if they select USB as the connection type.
2. When setting up Wi-Fi, if a mistake is made in entering the access point credentials and station mode is selected, then the credentials can't be changed without connecting via USB (or nuking the flash and reinstalling the firmware). So without USB support this can be very annoying. However, there will soon be a firmware update which will allow the Pico W to be forced to boot in AP mode (by pulling a GPIO low).

When/if I implement these features then I'll remove the USB requirement.

**Comment by matemaciek** [comment]
One reasonable scenario is that someone sets up WiFi data via USB from phone, and then uses it on Chromebook via WiFi, as it has nice big screen.

Have you considered opening the protocol? WiFi connection suggests possibility of developing platform independent web-based client.
